recipes/kuba-zip/all/conanfile.py

```diff
@@ -0,0 +1,61 @@
+import os
+from conans import ConanFile, CMake, tools
+
+
+class ZipConan(ConanFile):
+    name = "kuba-zip"
+    license = "Unlicense"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/kuba--/zip"
+    description = "A portable, simple zip library written in C"
+    topics = ("conan", 'zip', 'compression', 'c', 'miniz', 'portable', 'hacking')
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False],
+               "fPIC": [True, False]}
+    default_options = {"shared": False,
+                       "fPIC": True}
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake",
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        cmake = CMake(self)
+        cmake.definitions["CMAKE_DISABLE_TESTING"] = True
+        cmake.definitions["ZIP_STATIC_PIC"] = self.options.get_safe("fPIC", False)
```

Comment:
Could we ask the upstream for a new release?
This way we will remove small windows change in our cmake and `ZIP_STATIC_PIC` will work properly.

Reply:
It's deleted on Linux to assume always true =)

```suggestion
        cmake.definitions["ZIP_STATIC_PIC"] = self.options.get_safe("fPIC", True)
```